File: api-tester/src/main/java/com/revenuecat/apitester/java/VerificationResultAPI.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.VerificationResult;

@SuppressWarnings({"unused"})
final class VerificationResultAPI {
    static void check(final VerificationResult verificationResult) {
        switch (verificationResult) {
            case VERIFIED_ON_DEVICE:
            case NOT_REQUESTED:
            case VERIFIED:
            case FAILED:
        }
    }
}

File: api-tester/src/main/java/com/revenuecat/apitester/kotlin/VerificationResultAPI.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.VerificationResult

@Suppress("unused", "UNUSED_VARIABLE")
private class VerificationResultAPI {

    fun check(verificationResult: VerificationResult) {
        when (verificationResult) {
            VerificationResult.VERIFIED_ON_DEVICE,
            VerificationResult.NOT_REQUESTED,
            VerificationResult.VERIFIED,
            VerificationResult.FAILED,
            -> {}
        }.exhaustive
    }
}
